### DISCUSSION TITLE
Small visual nit-pick - nested groups missing padding

### DISCUSSION THREAD

**Discussion by STR1234** [payloadcms/payload#2250] [Feature Requests & Ideas]
# Bug Report

Nested groups show no padding between each other.

The structure on the image is:

```
group (Social Media Section) {
  - group (Facebook)
  // missing padding here :P
  - group (Instagram)
}

```
## Steps to Reproduce

1. Create a group that consists of two groups.
2. Miss padding between the nested groups
     in the Admin UI. :'(

## Other Details

<img width="1020" alt="Screenshot 2023-03-04 at 17 54 50" src="https://user-images.githubusercontent.com/9266694/223232540-448c000a-e648-44fa-9ec3-c78e8684b913.png">

**Comment by jmikrut** [comment]
Hey @STR1234 — this is actually working as intended, because the top-level group spacing is quite large and can get unwieldy when you have multiple levels of groups all nested within one another.

But, what is it exactly that you'd like to see here? More top + bottom spacing? You can add that via custom styles on your group config, but there's always a chance we can tweak our internal default styles to align more closely with your expectations.

As this is not something actionable at the moment, I'll convert this to a Discussion and we can keep the conversation going there. 👍

**Reply by STR1234** [reply]
Thanks for the fast reply :) What I would like to see in the example is more bottom spacing underneath the facebook group. I can however understand your thought process behind the decision to have it as it is right now and I will look into custom styling the groups to meet our expectations.
Thanks again :)
